src/test/java/xogame/board_utils/GeneratorTest.java

```diff
@@ -0,0 +1,78 @@
+package xogame.board_utils;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import org.junit.jupiter.api.Test;
+
+import xogame.models.Board;
+import xogame.utils.Result;
+
+public class GeneratorTest {
```

Comment:
We keep talking about boards here, yet all I see from the code are 1D arrays of integers. I think the naming here is not adequate.

Reply:
Change the name of the class to something more specific.